client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -133,6 +154,7 @@ export const getTask = (
 					actionDispatchArgs: [ siteId, task.id ],
 				} ),
 				isSkippable: true,
+				...jetpackAppBanner,
```

Comment:
I appreciate the feedback, I can see how the code was a bit confusing and like your approaches for making it clearer. I've gone with the second option, to avoid mutating: https://github.com/Automattic/wp-calypso/pull/68511/commits/7badcdaca1aa03eb137da7a1ec3f524c4b3e5723